subsys/net/lib/nrf_cloud/src/nrf_cloud_fota_common.c

```diff
@@ -258,13 +258,13 @@ int nrf_cloud_fota_fmfu_apply(void)
 	err = fmfu_fdev_load(fmfu_buf, sizeof(fmfu_buf), fmfu_dev.dev, fmfu_dev.offset);
 	if (err != 0) {
 		LOG_ERR("Failed to apply full modem update, error: %d", err);
-		(void)nrf_modem_lib_init();
```

Comment:
i think the intent here was to return the modem to a functional state if an update failed.
would calling  `nrf_modem_lib_shutdown();` before `nrf_modem_lib_init` accomplish that?

Reply:
Yeah, that should do it.